cmd/template/advanced/files/htmx/efs.go.tmpl

```diff
@@ -2,5 +2,5 @@ package web
 
 import "embed"
 
-//go:embed "js"
+//go:embed "assets"
```

Comment:
doesn't this needs to be assets/*

Reply:
@H0llyW00dzZ, it sounds great!, but for now the reason is to help user who wants to add a simple custom css or add some images with just putting their folder into `assets/` and load it in their templ. :heart: 
  
@EssaAlshammri, I've test it with generating all the go-blueprint supported frameworks + htmx and the htmx is still loads and works. For your suggestion, I think I need to test it more further. Thx for reviewing my code btw. :+1:

Reply:
> doesn't this needs to be assets/*

@EssaAlshammri 

I've added more test stuff in file server like css file and img file with `go:embed "assets"` directive, it looks fine. Then I reread the embed pkg documentation, interestingly I found:

> If a pattern names a directory, all files in the subtree rooted at that directory are embedded (recursively), except that files with names beginning with ‘.’ or ‘_’ are excluded.

Should I update to `assets/*` ?

Reply:
> @H0llyW00dzZ, it sounds great!, but for now the reason is to help user who wants to add a simple custom css or add some images with just putting their folder into `assets/` and load it in their templ. ❤️
> 
> @EssaAlshammri, I've test it with generating all the go-blueprint supported frameworks + htmx and the htmx is still loads and works. For your suggestion, I think I need to test it more further. Thx for reviewing my code btw. 👍

@kobamkode
How can you design the CSS for htmx? I am so bad at designing websites, to be honest.

Reply:
yeah the way you did it was right

I'm sorry it didn't work for me at first and worked when added * 

but now everything is working as expected 

yeah I'm using it for tailwind styles

![image](https://github.com/Melkeydev/go-blueprint/assets/10750698/aea3c16d-07cd-49f7-9141-608b7c117722)